streamz/dataframe/tests/test_cudf.py

```diff
@@ -0,0 +1,405 @@
+from __future__ import division, print_function
```

Comment:
I appreciate the large number of tests.

The xfails are here to indicate an intention to fix in the future? Do they depend on changes upstream?

Could it be possible to combine pandas and cudf tests to avoid duplication? Is that desirable?

Reply:
These xfails are due to cudf functions/methods not implemented yet, but will be available soon. So, these does not depend on upstream. 
Eventually we would want to combine these two. Matt suggested me to use a separate file for tests as most of pandas tests fail with cudf.

Reply:
OK, that's fair enough, so long a it's a conscious decision. Maybe worth putting in a comment at the top of the module.